src/ConnectionAutomaton.cpp

```diff
@@ -260,6 +277,15 @@ void ConnectionAutomaton::closeStreams(StreamCompletionSignal signal) {
 }
 
 void ConnectionAutomaton::processFrame(std::unique_ptr<folly::IOBuf> frame) {
+  auto thisPtr = this->shared_from_this();
+  auto movedFrame = folly::makeMoveWrapper(frame);
```

Comment:
this is making work for us to revert, I'll try to see if I can fix travis for 4.9